modules/core/src/shaderlib/index.js

```diff
@@ -47,4 +48,4 @@ export function createProgramManager(gl) {
   return programManager;
 }
 
-export {picking, project, project32, gouraudLighting, phongLighting, shadow};
+export {picking, project, project32, projectPosition, gouraudLighting, phongLighting, shadow};
```

Comment:
Ok I thought this function was exposed via `project.projectPosition` but apparently not. I think we removed it when we moved the functionality to `layer.projectPosition`.

I prefer to avoid exposing another API. Maybe add a second parameter to `layer.projectPosition` to allow overriding the project options?